File: falcon/test_falcon.py
# -*-coding:utf-8-*-
# author:liushangliang@xunlei.com

import time

import falcon

payload = [{
    "endpoint": "test-endpoint",
    "metric": "test-metric",
    "timestamp": int(time.time()),
    "step": 60,
    "value": 1,
    "counterType": "GAUGE",
    "tags": "role=slave"
}]

if __name__ == "__main__":
    falcon.report(payload)

File: falcon/test_falcon.lua
-- -*- coding:utf-8 -*-
-- author:liushangliang@xunlei.com

package.cpath = package.cpath .. ";/usr/lib/lua/5.1/?.so;"
local payloads = {
    {
        endpoint= "test-endpoint",
        metric= "test-metric",
        timestamp= os.time(),
        step= 60,
        value= 1,
        counterType= "GAUGE",
        tags= "idc=lg,loc=beijing",
    }
}

local falcon = require("falcon")
falcon.static(payloads)
